cmd/openshift-tests/upgrade.go

```diff
@@ -77,6 +77,8 @@ func upgradeTestPreTest() error {
 	parseUpgradeOptions(opt.TestOptions)
 	upgrade.SetToImage(opt.ToImage)
 	switch opt.Suite {
+	case "none":
+		return filterUpgrade(upgrade.MinimalTests(), func(string) bool { return true })
```

Comment:
Wait... none would be no tests.  None should be "zero things get run".  You still need to skip some tests for single mode (in the test, not here).

Reply:
Ah, sure. Lets start with none (mostly for paused workers) / minimal (SNO) suite

Reply:
I would prefer you don't add minimal though - that's what I'm specifically asking you not to do :)  To fix the tests.

As far as filtering, it was intended that you could filter test names with --dry-run and grep and then -f -, but it looks like that is broken for the upgrade subset tests.

Reply:
I would prefer to not run the invariants for upgrade for now, so none is probably sufficient for SNO

Reply:
Remove minimal, keeping none